frontend/src/modules/Servicemap/utils.ts

```diff
@@ -123,9 +125,9 @@ export const getTooltip = (link: {
 							</div>`;
 };
 
-export const transformLabel = (label: string) => {
-	const MAX_LENGTH = 13;
-	const MAX_SHOW = 10;
+export const transformLabel = (label: string, zoomLevel: number) => {
+	const MAX_LENGTH = 13 * (zoomLevel / 0.9);
```

Comment:
It would be helpful to add a comment in code to explain the user arbitrary numbers.

Reply:
Added a comment